docs/developer-guide/etendo-classic/concepts/data-model.md

```diff
@@ -0,0 +1,513 @@
+
+In this header, it is possible to select **Display value** field. In case this is selected, the list items will be represented by concatenating their value and their name, if not, only name will be displayed.
+
+The next step is to add values to the list. To do so, go to **List Reference** tab, and create the values you need. Here there are three important fields to take into account:
+
+- **Search Key** this is the value (not visible in UI) that the column using this reference will take when it is selected.
+- **Name** it is the name that will be displayed in the drop down list.
+- **Sequence Number** it is the position of the element when the drop down list is rendered. This field is nullable, depending on the values it has in the elements of the list, the sorting is done as follows:
+  - If the value of this field is null for all the elements in the list, the list will be sorted alphabetically.
+  - If all the elements have this value set, they will be ordered taking it into account.
+  - If some of the elements have it and some have not, the ones having it will be shown at the beginning of the list sorted by this number, the rest of them will be shown at the end ordered alphabetically.
+
+Just an additional consideration, it is possible to add elements in a module different to its header one, in this case the **Search Key** for the values must start with the module's DBPrefix to avoid naming clashes.
+
+##### ID
+
+Etendo uses UUID as primary key for its tables. The standard name for a primary key column is **TableName_ID** , so for example for **HR_Salary** table its primary key column would be name **HR_Salary_ID**.To be able to store UUIDs the physical type for the column must be **VARCHAR2(32)**. In addition, that column must be set as primary key in database.
+
+**ID** reference is used for primary key columns. Usually, it should not be displayed to the user because this key is senseless for them (use identifiers instead), but if it shown the UI for this reference is like **String** reference, a one-line textbox.
+
+##### TableDir
+
+**TableDir** reference stands for **Table Direct**. As explained in the previous section the standard name for a primary key column is **TableName_ID**, in the same way the standard name for a foreign key column is the name of the foreign primary key column. For example if we create a new table named **HR_MyTable** and we want add a foreign key to _HR_Salary_ , the column should be named **HR_MyTable.HR_Salary_ID**. If this naming rule is followed the **Create columns from DB** process will set this new column as **TableDir** reference. When the UI for this column is created a drop down list is generated with all the elements in the foreign table that can be referred from the current record. If the column name is not the same of the foreign key table's primary key column's name TableDir reference will not work, it will be necessary to define a **Table** reference for that column.
```

Comment:
```suggestion
`TableDir` reference stands for **Table Direct**. As explained in the previous section the standard name for a primary key column is `TableName_ID`, in the same way the standard name for a foreign key column is the name of the foreign primary key column. For example if we create a new table named `HR_MyTable` and we want add a foreign key to `HR_Salary` , the column should be named `HR_MyTable.HR_Salary_ID`. If this naming rule is followed the **Create columns from DB** process will set this new column as **TableDir** reference. When the UI for this column is created a drop down list is generated with all the elements in the foreign table that can be referred from the current record. If the column name is not the same of the foreign key table's primary key column's name `TableDir` reference will not work, it will be necessary to define a **Table** reference for that column.
```